README.md

```diff
@@ -1 +1,39 @@
-# Concert-Page
\ No newline at end of file
+
+# CONCERT PAGE
+- I build an online website for a conference which is a concert.
+- I implemented javascript features where a section of html was created dynamically by javascript code.
+## Built With
+
+- HTML/CSS linters
+- JavasScript
+- media queries
+
+# Live Demo link https://kelvin-ben.github.io/Concert-Page/
+
+👤 **Kelvin Ben**
+
+- GitHub: [@Kevo-hacker] (https://github.com/Kevo-hacker)
+- Twitter: [@kevoyout144] (https://twitter.com/kevoyout144)
+- LinkedIn: [Kelvin Ben] (https://www.linkedin.com/in/kelvin-ben-323043173/)
+
+👤 
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+- Thank you Microverse for the chance to learn through others
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] Kindly gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in the README file.